docs/docs-beta/docs/dagster-plus/insights.md

```diff
@@ -1,5 +1,157 @@
 ---
 title: "Insights"
+description: "Visiblity into historical usage, cost, and metadata."
 ---
 
-# Dagster+ Insights
\ No newline at end of file
+# Dagster+ Insights
+
+Using Dagster+ Insights, you can gain visibility into historical usage and cost metrics such as Dagster+ run duration, credit usage, and failures.
+Visualizations are built right into the Dagster UI, allowing you to explore metrics from Dagster as well as downstream systems (like Google BigQuery) all in one place.
+
+### With Insights, you can
+
+- [Explore usage trends in your Dagster pipelines](#explore-dagsters-built-in-metrics)
+- [Integrate additional metrics](#integrate-other-metrics), like data warehouse cost or your own custom metadata
+- [Export metrics](#export-metrics) from Dagster+
+- [Create alerts](/dagster-plus/deployment/alerts#alerting-when-an-insights-metric-crosses-a-threshold) based off of Insights metrics {/* TODO: write this alerts section) */}
+
+<details>
+  <summary>Prerequisites</summary>
+
+To use Insights, you'll need a Dagster+ account.
```

Comment:
Does it make sense to include this in Dagster+ content or is it implied?

Reply:
You'd be surprised how many folks look at Dagster+ content and think it also applies to OSS. I say leave it.

Reply:
doessnt hurt in case someone stumbles in on search